### PR TITLE
adding new distribution

### PR DESCRIPTION
**Pull request by jessicanicholson**

### CODE REVIEW

File: distributions/Dist_jn1348.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+class Dist_jn1348(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = np.pi/-2.0
+		self.x_max = np.pi/2.0
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.sin(x)
```
Comment (cranmer) [inline distributions/Dist_jn1348.py:14]: PDFs must be positive

Comment (jessicanicholson) [inline distributions/Dist_jn1348.py:14]: Hi Professor Cranmer,

I fixed my error, but do I need to submit another pull request because that
button isn't popping up like it did before.

Best,
Jessica Nicholson

**Comment by cranmer** [comment]
merged
